javascript/javascript2/week1/week1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Week1 Homework</title>
+    <link rel="stylesheet" href="week1.css" />
+  </head>
+  <body>
+    <div class="Tasks">
+      <h1 class="headlines">First task</h1>
+      <p>
+        Find the ShortestWord in the array ["bil", "plante", "kaffe", "bog",
+        "ø", "planetarium"]
+      </p>
+      <p id="ShortestWord"></p>
+    </div>
+    <div class="Tasks">
+      <h1 class="headlines">Second task</h1>
+      <p>
+        We have to count danish letters in a string.<br />
+        Given String:"Blå grød med røde bær"
+      </p>
+      <p id="danishLetters"></p>
+    </div>
+    <div class="Tasks">
+      <h1 class="headlines">Third task</h1>
+      <fieldset>
+        <center>
+          <h2>Spirit animal name generator</h2>
+          <input type="text" id="name" /> <br />
+          <br /><button id="getname" type="submit">Get full Name</button><br />
+          <h1 id="animal"></h1>
+          <br />
+        </center>
+      </fieldset>
+    </div>
+    <script src="shortword.js"></script>
+    <script src="count.js"></script>
+    <script src="animalname.js"></script>
+     <center>
+    <div class="hyfbay">
+    <main>
+
+    <div class="Tasks">
+      <h1 class="headlines">hyfBayHelpers</h1>
+      <script src="hyfBayHelpers.js"></script>
+      <script src="main.js"></script>
+    </div></main></center>
+  </body>
+</html>
```

Comment:
i want to see the preview of both of the html files, in order to see if all the things got rendered accordingly. Maybe if possible for you , please make a link for these files and paste it here in the  reply of these comments so i can have a look at them as well. Anyway keep the good work up, you're doin the great job!

Reply:
https://github.com/thathinisai/hyf-homework/pull/7/files#diff-c16e91587b915f0ddb21b50861552b9c97da89928acfed7235affb9378eec8e8